### ISSUE TITLE
Compile Errors

### ISSUE THREAD

**Issue by pylang** [ihrke/markdown-paper#1]
Thank you for this contribution.  It is helpful to have an example that builds academic pdfs from  markdown.  

### Issues

There are a few issues I got when running the `make` file.  Some issues (1, 2) were somewhat resolved, but I bring them to your attention:

1.  `-S` is deprecated:

![2018-06-13 14_26_19-makefile - markdown-paper-master - visual studio code](https://user-images.githubusercontent.com/10778668/41370577-077e7f5e-6f16-11e8-9fcb-bc5ff458de5d.png)

(Resolved) Replaced `-S` with `-f markdown+smart -t markdown-smart` as instructed

2. Dependencies can't be found:

![2018-06-13 14_30_29-makefile - markdown-paper-master - visual studio code](https://user-images.githubusercontent.com/10778668/41370733-660f2578-6f16-11e8-81b6-ac4cf60ed1b0.png)

(Resolved) Replaced lines 8 & 9 in `Makefile` with hardlinks to the dependencies directories:

![2018-06-13 14_32_06- makefile - markdown-paper-master - visual studio code](https://user-images.githubusercontent.com/10778668/41371229-b594924e-6f17-11e8-9e72-79579a436ad9.png)

3.  Footnotes and figure numbers give the following errors during compilation (I pressed <kbd>Enter</kbd> to continue):

Footnotes
![2018-06-13 14_36_46-makefile - markdown-paper-master - visual studio code](https://user-images.githubusercontent.com/10778668/41371301-e8ceea1a-6f17-11e8-8bc0-62a69ef5e381.png)

Figure numbers
![2018-06-13 14_37_41-makefile - markdown-paper-master - visual studio code](https://user-images.githubusercontent.com/10778668/41371457-5eb69be2-6f18-11e8-9917-63779367d642.png)

**(Unresolved)**: please advise

### Summary

Issues 1-2. were somewhat resolved.  I was unable to resolve the last issue (3).  
- I'm not sure why footnotes aren't working in this repo.  They work elsewhere.  
- It seems the `pandoc-fignos` filter may not be pre-processing the markdown file.   I know `pandoc-fignos` works on other files.

I would appreciate your advice on how to proceed.  This is a great tool I'd like to adapt to my  workflow.  Thank you.

### System 

Windows 7 | Gnu Make 3.80 | MiKTeX 2.9.6630 | pdfTeX 3.14159265-2.6-1.40.19 | pandoc 2.2.1 | pandoc-fignos 1.2.0

**Comment by pylang** [comment]
The great work in [this issue](https://github.com/tomduck/pandoc-fignos/issues/49) resolved the fig number issues (item 3 above).    

Given the updated `pandoc-fignos` (v 1.3.0), figures and fig numbers now appear as expected.

![2018-08-09 13_47_24-paper pdf - foxit reader](https://user-images.githubusercontent.com/10778668/43916207-ea3d438e-9bda-11e8-87c7-e2491466b273.png)
![2018-08-09 13_48_04-paper pdf - foxit reader](https://user-images.githubusercontent.com/10778668/43916208-eb8b5a28-9bda-11e8-91ed-65212e648eef.png)




 provided the Makefile is edited as follows:

- `./templates/dependencies/` is substituted (lines 8 and 9)
- `-S` is replaced with `-f markdown+smart` (lines 18)

Call the following to rebuild the `paper.pdf`:
```bash
> make -B
```
---
Another issue arises concerning the references.  They remain unnumbered (notice the question marks):

![2018-08-09 13_36_43-paper pdf - foxit reader](https://user-images.githubusercontent.com/10778668/43916301-2fba5f78-9bdb-11e8-80a6-83825bbcbba1.png)

Here is the error:

![2018-08-09 13_33_13-cmder](https://user-images.githubusercontent.com/10778668/43916313-3621dfa8-9bdb-11e8-8b89-0ed8047f511b.png)

Any thoughts on why the references are not working?

**Comment by ihrke** [comment]
Hi,

sorry for the late reply, for some reason I did not see this message before.
Seems like you were able to resolve most of the problems for yourself. I have fixed the ´-S´ deprecation in the Makefile. Regarding your last error, it seems like bibtex is looking for the file `references.bib.bib`, so maybe try removing the `.bib` from `references.bib`? On my systems (mac and linux), this problem does not occur.

Best,

Matthias
